Sources/PackageModel/UserToolchain.swift

```diff
@@ -741,7 +741,11 @@ public final class UserToolchain: Toolchain {
             // targetInfo from the compiler
             let targetInfo = try customTargetInfo ?? Self.getTargetInfo(swiftCompiler: swiftCompilers.compile)
             self._targetInfo = targetInfo
-            triple = try swiftSDK.targetTriple ?? Self.getHostTriple(targetInfo: targetInfo, versioned: false)
+            var defaultTriple = try Self.getHostTriple(targetInfo: targetInfo, versioned: false)
+            if defaultTriple.isOpenBSD() {
```

Comment:
This triple was always versioned before my PR to fix the deployment target issues on Darwin, so we may want to just make this an is-non-darwin check to preserve behavior, wdyt?

Reply:
Done.